### PR TITLE
Roadmap

### PR DESCRIPTION
**Pull request by sashko**

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -20,9 +20,6 @@ The PELUX project aims to provide biannual releases. The first release was done
 in January 2018. For already released versions, see the [downloads](/downloads)
 page.
 
-### PELUX 1.1 [April 2018]
-Bugfix release
-
 ### PELUX 2.0 [June 2018]
```
Comment (AshleyNair) [inline roadmap.md]: Release date for PELUX 2.0 should be July 2018

File: roadmap.md [hunk 2]
```diff
@@ -20,9 +20,6 @@ The PELUX project aims to provide biannual releases. The first release was done
...
 Content of this release is under discussion. Tentatively the topics in focus will be:
 - Intel ARP board support
```
Comment (AshleyNair) [inline roadmap.md]: I am not sure if the NVIDIA board is supported

Comment (sashko) [reply roadmap.md]: it is apparently not

Comment (AshleyNair) [reply roadmap.md]: Then we should remove that point as well

**Pull request merged**